### DISCUSSION TITLE
Hello, I need help to resize widget and box

### DISCUSSION THREAD

**Discussion by Razer-0** [beeware/toga#2490] [Debugging help]
Hello, I need help to resize widget and box in my app taking up all the available space with toga, does anyone know how to do this, I checked the style sheet but it didn't give me any clue on how to do this, I need my app to fit the space in screen

**Comment by freakboy3742** [comment]
It sounds like you're looking for the `flex` option to the stylesheet. Toga's Pack layout is essentially a reduced version of CSS Flexbox; if you mark an element as having style `flex=1`, you're indicating you want that element to expand, with the relative rate of expansion being controlled by the value of the flex (so flex=2 will expand twice as much as flex=1, once all fixed size elements have had their size allocated. The flexible size is applied in the direction of the containing box - so a direction=COLUMN box will expand vertically, and a direction=ROW box will expand horizontally.

**Reply by Razer-0** [reply]
I understand then if flex = 1 , you are indicating that you want that element to expand, with the relative expansion rate controlled by the value of flex, so flex=2 will expand twice as much as flex=1, so for the example widget toga.Box() occupies the entire main_window the flex value of toga.Box() should be flex=2, and if I want to add toga.WebView() to the toga.Box() and occupies all the available space the property It should be flex=2, it would help me a lot if you could provide an example.

**Comment by Razer-0** [comment]
@freakboy3742 Thanks for your help, I was able to solve my problem and understand perfectly how the `flex `property works. Now my app can now resize the window, thank you.
